components/scream/src/physics/p3/p3_functions.hpp

```diff
@@ -221,14 +221,14 @@ struct Functions
     const view_1d_ptr_array<Spack, nfield>& V, // (behaviorally const)
     const view_1d_ptr_array<Spack, nfield>& r);
 
-  template <Int kdir, int nfield>
+  template <int nfield>
```

Comment:
Only offering this particular interface for now (unlike the three versions of calc_upwind).